services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/impl/SubnetService.java

```diff
@@ -85,7 +85,7 @@ public void buildSubnetStates(NetworkConfiguration networkConfig, UnicastGoalSta
             }
 
             Subnet.SubnetState.Builder subnetStateBuilder = Subnet.SubnetState.newBuilder();
-            subnetStateBuilder.setOperationType(networkConfig.getOpType());
+            subnetStateBuilder.setOperationType(Common.OperationType.INFO); // networkConfig.getOpType()
```

Comment:
please remove the commented out part.

actually, you don't even need to set it to INFO because it is the default based on your schema update. I suggest removing this to save this one line of execution :) and replace with a comment it is set as INFO by default.

Reply:
As discussed, let us create a standalone class to assign the OperationType in various scenarios, and pass the reference of the object to VPC/Subnet/.. services classes to consume the value. 

This way, we have a centralized place for field assignment and make it easier to change in the future.